indexing-hadoop/src/main/java/io/druid/indexer/path/MultiplePathSpec.java

```diff
@@ -0,0 +1,80 @@
+/*
+* Licensed to Metamarkets Group Inc. (Metamarkets) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. Metamarkets licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+package io.druid.indexer.path;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.api.client.repackaged.com.google.common.base.Preconditions;
+import io.druid.indexer.HadoopDruidIndexerConfig;
+import org.apache.hadoop.mapreduce.Job;
+
+import java.io.IOException;
+import java.util.List;
+
+public class MultiplePathSpec implements PathSpec
+{
+  private List<PathSpec> childPathSpecs;
+
+  public MultiplePathSpec(
+      @JsonProperty("childPathSpecs") List<PathSpec> childPathSpecs
```

Comment:
It's a little weird that this is called childPathSpecs but in the json it comes under something that is inputSpec. Maybe we can dodge the issue by calling it "children".